specification/header.adoc

```diff
@@ -59,7 +59,12 @@ Copyright 2023 by RISC-V International.
 
 [preface]
 include::contributors.adoc[]
-
 include::intro.adoc[]
+include::glossary.adoc[]
+include::arch_overview.adoc[]
```

Comment:
Where do we cover the "requirement" section? In "arch_overview"

Reply:
I added a few subsections in that file.